modules/installation/nav-installation-guide.adoc

```diff
@@ -4,8 +4,13 @@ ifndef::backend-pdf[]
 * Installation Guide
 ** xref:install-intro.adoc[Introduction]
 ** Requirements
+ifeval::[{suma-content} == true]
 *** xref:general-requirements.adoc[General Requirements]
 *** xref:hardware-requirements.adoc[Hardware Requirements]
+endif::[]
+ifeval::[{uyuni-content} == true]
+*** xref:uyuni-install-requirements.adoc[Requirements]
+endif::[]
 *** xref:network-requirements.adoc[Network Requirements]
 **** xref:ports.adoc[Required Network Ports]
 *** xref:client-requirements.adoc[Supported Clients]
```

Comment:
This document also should be reviewed as it has:
> In this table,  indicates that clients running the operating system are supported by SUSE, and 
> indicates that it is not supported. Fields marked as  are under consideration, and may or may not be
> supported at a later date.

> Public clouds provide Uyuni under a Bring Your Own Subscription (BYOS)
> 
>  Center. model. This For more means information that you must about register registering instances Uyuni with with the SUSE SUSE Customer Customer Center, see [ Installation › General-requirements › ].

Reply:
added